frontend/.prettierrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "printWidth": 80,
```

Comment:
I usually find 80 too low, I set mine to black (python formatter) to 120. Biased because I'm working on dual displays so lots of space, though. Thoughts on 120?

Reply:
80 is the standard so I just went with it, we can increase it to 120 for more clarity if required